model/user.go

```diff
@@ -406,11 +406,11 @@ func (u *User) AddNotifyProp(key string, value string) {
 }
 
 func (u *User) GetFullName() string {
-	if u.FirstName != "" && u.LastName != "" {
+	if len(u.FirstName) > 0 && len(u.LastName) > 0 {
```

Comment:
Why change these to use len? As far as I can tell, this is equivalent to what we had before

Reply:
true it is equivalent but by some strange reason it wasn't working, dunno the reason, but as soon as I did this change every option started working as expected